Modules/_cursesmodule.c

```diff
@@ -1217,8 +1221,9 @@ PyCursesWindow_GetStr(PyCursesWindowObject *self, PyObject *args)
         Py_END_ALLOW_THREADS
         break;
     case 1:
-        if (!PyArg_ParseTuple(args,"i;n", &n))
+        if (!PyArg_ParseTuple(args, "i:getstr", &n)) {
```

Comment:
It doesn't directly related to error messages in case of integer overflow. Remove all similar changes. They will be made by bpo-20171.